test/mountain_view/presenter_test.rb

```diff
@@ -0,0 +1,48 @@
+require "test_helper"
+class InheritedPresenter < MountainView::Presenter
+  attributes :title, :description
+  attribute :data, default: []
+
+  def title
+    "Foo#{properties[:title].downcase}"
+  end
+end
+
+class MountainViewPresenterTest < ActiveSupport::TestCase
+  def test_partial
+    presenter = MountainView::Presenter.new("header")
+    assert_equal "header/header", presenter.partial
+  end
+
+  def test_locals
+    properties = {name: 'Foo', title: 'Bar'}
+    presenter = MountainView::Presenter.new('header', properties)
+    assert_equal add_properties(properties), presenter.locals
+  end
+
+  def test_attributes
+    properties = {name: 'Foo', title: 'Bar'}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.